plugins/woocommerce/tests/e2e-pw/tests/shopper/cart-checkout-restricted-coupons.spec.js

```diff
@@ -34,6 +34,27 @@ test.describe(
 				version: 'wc/v3',
 			} );
 
+			// make sure the store address is US
+			await api.post( 'settings/general/batch', {
+				update: [
+					{
+						id: 'woocommerce_store_address',
+						value: 'addr 1',
+					},
+					{
+						id: 'woocommerce_store_city',
+						value: 'San Francisco',
+					},
+					{
+						id: 'woocommerce_default_country',
+						value: 'US:CA',
+					},
+					{
+						id: 'woocommerce_store_postcode',
+						value: '94107',
+					},
+				],
+			} );
```

Comment:
This is to ensure the store is always US. I experienced at first Afghanistan, might be some leftover from the other tests. Let's keep this for the test to ensure US address.